src/theme_contemporary_sticky_header.css

```diff
@@ -0,0 +1,152 @@
+@font-face {
```

Comment:
Do you think you can help me doing that? I am new to github and I don't really know how to make that change available to everyone. I think it would be a good addition.